crates/astria-proto/README.md

```diff
@@ -1,24 +1,18 @@
-# Astria RPC
+# Astria Proto
 
-This repo contains all the protobuf definitions for the different Astria RPC services.
+This repo contains all the protobuf packages for Astria. 
 
 ## Protos and Buf Build
 
 [Buf Build](https://buf.build/) is a platform and registry for sharing Protocol Buffers between team members. It also comes with a set of tools to generate gRPC servers and clients in a range of languages.
 
-[Astria's Buf Build organization](https://buf.build/astria)
+[Astria's published protos](https://buf.build/astria/astria)
 
-## Adding an RPC service
+## Adding a package
 
-* create new cargo lib in root of repo, e.g. `cargo new sandwich-apis --lib`
-* add the new lib to `Cargo.toml` as a workspace member
-* define protos in `sandwich-apis/proto/`
-* create `buf.gen.yaml` and `buf.yaml` in `sandwich-apis/`.
-* use `execution-apis/` as a reference for `buf.gen.yaml` and `buf.yaml`.
-* create `build.rs`, `touch sandwich-apis/build.rs`
-* copy over logic from `execution-apis/build.rs` for now, updating necessary paths
-* update `sandwich-apis/src/lib.rs` to include the files generated from `build.rs`. see `execution-apis/src/lib.rs`
-* add steps to `.github/workflows/pull-request.yml` and `.github/workflows/push.yml` to lint, build, and push the buf image from the new proto defs
+* Create a new folder under `proto/astria/{new_package}`
+* write protos in new folder using package name `astria.{new_package}`
+* update `src/lib.rs` to build a rust module from your new defintions
```

Comment:
it's actually the build script (build.rs) that's doing the building, lib.rs just exports the rust artifacts of building the protos, so maybe reword this to `update `src/lib.rs` to include your new module`?